content/_data/books/Mastering_Jenkins.yml

```
@@ -0,0 +1,19 @@
+title: "Mastering Jenkins"
+url: https://www.amazon.in/Mastering-Jenkins-Jonathan-McAllister/dp/1784390895/ref=sr_1_6?crid=3CUHJHQY7X09V&keywords=jenkins+books&qid=1676393236&sprefix=jenkins+books%2Caps%2C241&sr=8-6
```

I think it is better to link to the worldwide Amazon site and rely on their worldwide site to redirect the user to their local site.
```suggestion
url: https://www.amazon.com/Mastering-Jenkins-Jonathan-McAllister/dp/1784390895
```

---

@MarkEWaite Thanks for the guidance. I will make the necessary changes and will update my PR.